.github/workflows/pr_label.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
   label:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/labeler@v4
+      - uses: actions/labeler@v5
```

Comment:
There are [breaking changes](https://github.com/actions/labeler?tab=readme-ov-file#breaking-changes-in-v5) that require updating some configs